GameTest.java

```diff
@@ -0,0 +1,215 @@
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class GameTest {
+
+	Game game;
+	
+	@Before
+	public void setup(){
+		game = new Game();
+	}
+	
+	// In each test which checked the moving function, I asserted both changing places
+	
+	// - Pacman -
+	// Right
+	
+	@Test
+	public void PacmanRightWall(){
+		// arrange
+		this.game.board = new Board(new String[]{"PW"}).getBoard();
+		Location l = new Location(0, 0);
+		
+		// action
+		game.movePacman(l, Game.Directions.Right);
+		
+		// assert
+		assertEquals(Board.Creatures.Pacman, this.game.board[l.getX()] [l.getY()]);
```

Comment:
Again, don't use the internal "board" member directly. 
Ask for the content of a specific location - `this.game.getCreatureAt(location)` or `this.game.getCreatureAt(x,y)`